packaging/tc-image/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM receptor:latest
+
+RUN dnf install tc -y
+
+ENTRYPOINT ["/bin/bash"]
```

Comment:
It probably doesn't matter for this use case, but in a Receptor container that's going to be running a busy Workceptor, we expect it to be exec-ing lots of worker processes, including processes that themselves exec other processes ... which is exactly the situation where we really want our pid 1 to be an init process that does proper zombie process reaping.  This is why I've got the entrypoint running dumb-init in the Receptor container.

Reply:
the reason im using bash is because i want to redirect stdout and stderr, as far as i can tell i can't do that with `dumb-init`, if you know a way to do that with `dumb-init` i can switch to using that over bash

Reply:
I don't think there's a way.  We aren't likely to be running heavy Workceptor workload in this container though.